src/taskpane/taskpane.html

```diff
@@ -346,54 +468,88 @@ <h4>2. Choose which mode to use</h4>
                     <p><small>To <b>associate artifacts</b>, find the column of the artifact type you want to associate
                             to (e.g.: "Linked Requirement") and enter the ID(s) of the artifact(s) to associate to. You
                             can associate multiple artifacts at a time using a comma-separated list of IDs, e.g.:
                             335,336,337. It's possible to associate: </small></p>
                     <ul>
-                        <small>
-                            <li>Requirements to Requirements</li>
-                        </small>
-                        <small>
-                            <li>TestCases to Requirements</li>
-                        </small>
-                        <small>
-                            <li>TestCases to Releases</li>
-                        </small>
-                        <small>
-                            <li>TestCases to TestSets</li>
-                        </small>
+                            <li><small>Requirements to Requirements</small></li>
+                            <li><small>TestCases to Requirements</small></li>
+                            <li><small>TestCases to Releases</small></li>
+                            <li><small>TestCases to TestSets</small></li>
                     </ul>
                     <p><small>To <b>create a new comment</b>, simply enter the text under the column "New Comment". When
                             you send to / update Spira, this text will
                             be saved as a new comment in the artifact.</small></p>
                     <p>In case you have more than one sheet in your spreadsheet, make sure that the one you want to use
                          the add-on with is ordered as the first one. </p>
-
-
-
-
-
-
-
-
-
+                    </p>
+                </li>
+                <li>
+                    <p>
+                        <b>Logout</b>
+                        <small>Close your connection with Spira and take you back to the add-in's login page</small>
+                    </p>
+                </li>
+            </ul>
+            <p>If you are a System Administrator in Spira, you will have access to the following extra functionalities:</p>
+            <ul class="list">
+                <li>
+                    <p>
+                        <b>Add new users to Spira</b>
+                        <small>Select this under "Operations" and click on "Prepare Data Template Sheet" to load the template. 
+                            Then, populate the sheet with at least the required (bold) fields and hit "Send Data". 
+                            The just-created new users don't need to be approved in Spira. Please note that if an already existent 
+                            username is attempeted creation, no data will actually be created or updated in Spira and the username's ID will be shown in the first column.</small>
+                    </p>
```

Comment:
These each seem to have an extra closing </p> tag (starting with the closing tag on line 488) - I don't think it causes any issues as browsers just throw out closing tags that don't close anything, but is not good practice and should probably be removed just in case.